pkg/storage/cloudimpl/file_table_storage.go

```diff
@@ -41,6 +41,23 @@ func makeFileTableStorage(
 		return nil, errors.Errorf("FileTable storage requested but username or qualified table name" +
 			" not provided")
 	}
+
+	// FileTableStorage is not backed by a file system and so the name of the file
```

Comment:
@knz @dt moved the check to the server-side.  Also added a couple of tests in `userfile_test.go`.